test/fw/ptl/utils/plugins/ptl_test_loader.py

```diff
@@ -190,7 +195,20 @@ def prepareTestLoader(self, loader):
         old_loadTestsFromNames = loader.loadTestsFromNames
 
         def check_loadTestsFromNames(names, module=None):
-            rv = old_loadTestsFromNames(names, module)
+            tests_dir = names
+            if not self.testfiles:
+                self._du = DshUtils()
+                ptl_test_dir = self._du.which(exe='pbs_benchpress')
```

Comment:
what if pbs_benchpress is not in the path? you can use sys.argv[0].

Reply:
Can we use PBS_EXEC path and then replace pbs with ptl ?

Reply:
I don't think use PBS_EXEC is good. it's not guaranteed that use has installed PTL in /opt/ptl.
So it is best to use sys.argv[0]

Reply:
I agree. Wish we could have something called PTL_HOME like PBS_HOME which represents PTL installed path. 
For now, I have changed to sys.argv[0] considering user will always use full path of pbs_benchpress to run the test case.